src/harness/sas.py

```diff
@@ -142,6 +142,9 @@ def GetSasImplementationRecord(self, request, ssl_cert=None, ssl_key=None):
   def GetEscSensorRecord(self, request, ssl_cert=None, ssl_key=None):
     return self._SasRequest('esc_sensor', request, ssl_cert, ssl_key)
 
+  def GetFullActivityDump(self, ssl_cert=None, ssl_key=None):
+    return self._SasRequest('dump', None, ssl_cert, ssl_key)
```

Comment:
If you are planning to pass None in the _SasRequest, I think you need to update the definition of the _SasRequest and may be something like `'https://%s'%'/'.join(value for value in [self._base_url, self._sas_version, method_name, request] if value)` in the _RequestGet function